### ISSUE TITLE
Update title metadata and more so that test book passes Ace

### ISSUE THREAD

**Issue by ways2read** [daisy/epub-accessibility-tests#11]
The test book epub30-test-0300-20170809.epub has 104 serious violations according to Ace. This includes a11y metadata, contrast, aria roles and more.

**Comment by marisademeglio** [comment]
This may trigger retesting depending on if the changes would affect each test's <section> element contents.

**Comment by avneeshsingh** [comment]
Fundamental test book should achieve good results with Ace. At least it should not have serius issues.

**Comment by GeorgeKerscher** [comment]
We have no tested the titles with Ace and Epubcheck. The accessibility metadata has been added to all titles in the Early 2024 set of titles.
